tests/test_examples.py

```diff
@@ -575,6 +581,7 @@ class MultiCardSpeechRecognitionExampleTester(
     ExampleTesterBase, metaclass=ExampleTestMeta, example_name="run_speech_recognition_ctc", multi_card=True
 ):
     TASK_NAME = "regisss/librispeech_asr_for_optimum_habana_ci"
+    DATASET_NAME = os.environ.get("DATA_CACHE", 0)
```

Comment:
what happen if you don't set this environment?

Reply:
Verified working even without DATA_CACHE on a different machine. Both clip-roberta and speech-wav2vec2 work fine.